packages/universal-components/src/RangeDatePicker/NightsInDestination.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { Slider } from '../Slider';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+type Props = {|
+  +onNightsInDestinationChange?: ?(Array<number>) => void,
+  +nightsInDestination?: ?$ReadOnlyArray<number>,
+|};
+
+export default class NightsInDestination extends React.Component<Props> {
+  render() {
+    return (
+      this.props.nightsInDestination != null &&
+      this.props.onNightsInDestinationChange != null && (
+        <View style={styles.container}>
+          <View style={styles.container} />
+          <View style={styles.slider}>
+            <Slider
+              minValue={1}
+              maxValue={30}
+              startValue={this.props.nightsInDestination[0]}
+              endValue={this.props.nightsInDestination[1]}
+              step={1}
+              label="Nights in destination"
```

Comment:
We should pass this label as prop like in other components inside `universal`.

Reply:
Done